gcp_census/bigquery/bigquery_client.py

```diff
@@ -160,61 +142,31 @@ def get_table(self, project_id, dataset_id, table_id, log_table=True):
-            table_dict['tableReference']['projectId'],
-            table_dict['tableReference']['datasetId'],
-            table_dict['tableReference']['tableId'],
-            partition)
-
+    def stream_stats(self, row):
         insert_all_data = {
             'rows': [{
                 'json': {
-                    'snapshotTime': time.time(),
-                    'projectId': table_dict['tableReference']['projectId'],
-                    'datasetId': table_dict['tableReference']['datasetId'],
-                    'tableId': table_dict['tableReference']['tableId'],
-                    'creationTime': float(table_dict['creationTime']) / 1000.0,
-                    'lastModifiedTime': float(table_dict['lastModifiedTime'])
-                                        / 1000.0,
-                    'partition': partitions,
-                    'location': table_dict['location']
-                                if 'location' in table_dict else None,
-                    'numBytes': table_dict['numBytes'],
-                    'numLongTermBytes': table_dict['numLongTermBytes'],
-                    'numRows': table_dict['numRows'],
-                    'timePartitioning': table_dict[
-                        'timePartitioning'] if 'timePartitioning'
-                                        in table_dict else None,
-                    'type': table_dict['type'],
-                    'json': json.dumps(table_dict),
-                    'description': table_dict['description']
-                                   if 'description' in table_dict else None,
-                    'labels': [{'key': key, 'value': value} for key, value
-                               in table_dict['labels'].iteritems()]
-                              if 'labels' in table_dict else []
+                    row.data
                 },
-                'insertId': insert_id
+                'insertId': row.insert_id
             }]
         }
-        insert_all_response = self._stream_metadata(insert_all_data, partition)
+        insert_all_response = self._stream_metadata(insert_all_data,
+                                                    row.dataset_id,
+                                                    row.table_id)
         if 'insertErrors' in insert_all_response:
             logging.debug("Sent json: \n%s", json.dumps(insert_all_data))
             logging.error("Error during streaming metadata to BigQuery: \n%s",
                           json.dumps(insert_all_response['insertErrors']))
         else:
-            logging.debug("Stats have been sent successfully")
+            logging.debug("Stats have been sent successfully to %s.%s table",
+                          row.dataset_id, row.table_id)
 
     @retry(Error, tries=2, delay=2, backoff=2)
-    def _stream_metadata(self, insert_all_data, partition):
+    def _stream_metadata(self, insert_all_data, dataset_id, table_id):
+        partition = datetime.datetime.now().strftime("%Y%m%d")
```

Comment:
maybe utcnow() ?

Reply:
Yep, I'll fix it in the next pull request